protractor/e2e/integration/secong-page.spec.ts

```diff
@@ -59,4 +59,24 @@ describe('tests for second page', ()=> {
         expect(await secondPage.infoSubmitButton.isDisplayed()).toBeTruthy();
         await secondPage.infoSubmitButton.click();
     });
-});
+
+
+    fit('Homework 2: check amount of articles in the Blog', async () => {
+        await secondPage.blogHeaderButton.click();
+        expect (await secondPage.getCurrentUrl()).toContain('/blog.ng-book.com/');
+        expect (await secondPage.articles.count()).toEqual(20);
+        expect (await secondPage.articleHeader.count()).toEqual(20);
+        expect (await secondPage.articleDescription.count()).toEqual(20);
+        expect (await secondPage.articleLink.count()).toEqual(20);
```

Comment:
you can define amount of articles/headers/links once at the top of this it, like const expectedArticlesLength = 20; and reuse it